src/main/java/ngrok/configuration/NgrokConfiguration.java

```diff
@@ -26,7 +24,7 @@
     /**
      * Custom configuration path e.g.: '/home/user/example-directory/custom-config.yml'
      */
-    private String configPath;
+    private String config;
```

Comment:
I changed this to `config` because in the end this should be `ngrok.config` because in prvoius versions thats how this property was named, and this one was failing because of that https://github.com/kilmajster/ngrok-spring-boot-starter/blob/main/src/test/java/ngrok/runner/NgrokRunnerWithConfigFileTest.java

Reply:
Oh well, my bad. I was about to make this property name to be more clear, but hadn't keep in mind that the *interface* would change...